[2025-03-15T10:11:04.701+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-15T10:11:04.707+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: inference-pipeline.load_configuration manual__2025-03-15T10:11:02.424439+00:00 [queued]>
[2025-03-15T10:11:04.709+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: inference-pipeline.load_configuration manual__2025-03-15T10:11:02.424439+00:00 [queued]>
[2025-03-15T10:11:04.709+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-03-15T10:11:04.712+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): load_configuration> on 2025-03-15 10:11:02.424439+00:00
[2025-03-15T10:11:04.717+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=91) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-15T10:11:04.718+0000] {standard_task_runner.py:72} INFO - Started process 101 to run task
[2025-03-15T10:11:04.718+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'inference-pipeline', 'load_configuration', 'manual__2025-03-15T10:11:02.424439+00:00', '--job-id', '91', '--raw', '--subdir', 'DAGS_FOLDER/inference_pipeline.py', '--cfg-path', '/tmp/tmp9wgnwmfg']
[2025-03-15T10:11:04.719+0000] {standard_task_runner.py:105} INFO - Job 91: Subtask load_configuration
[2025-03-15T10:11:04.738+0000] {task_command.py:467} INFO - Running <TaskInstance: inference-pipeline.load_configuration manual__2025-03-15T10:11:02.424439+00:00 [running]> on host 55756404afe6
[2025-03-15T10:11:04.765+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='user' AIRFLOW_CTX_DAG_ID='inference-pipeline' AIRFLOW_CTX_TASK_ID='load_configuration' AIRFLOW_CTX_EXECUTION_DATE='2025-03-15T10:11:02.424439+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-15T10:11:02.424439+00:00'
[2025-03-15T10:11:04.766+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-15T10:11:04.786+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-15T10:11:04.791+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-15T10:11:04.791+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=inference-pipeline, task_id=load_configuration, run_id=manual__2025-03-15T10:11:02.424439+00:00, execution_date=20250315T101102, start_date=20250315T101104, end_date=20250315T101104
[2025-03-15T10:11:04.823+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-15T10:11:04.835+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-15T10:11:04.835+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
